AGENTS.md

```
@@ -35,6 +35,42 @@ Development setup, coding standards, and workflow guidelines. Essential for unde
 
 Reading these documents will save time and prevent common mistakes during development and deployment.
 
+## Critical Development Guidelines
+
+### ⚠️ IMPORTANT: Command Execution Rules
+
+**1. GitHub CLI Usage**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)